src/resources/triton_python_backend_utils.py

```diff
@@ -295,6 +295,128 @@ def triton_string_to_numpy(triton_type_string):
     return TRITON_STRING_TO_NUMPY[triton_type_string]
 
 
+def set_max_batch_size(config, value):
+    """Set the max batch size for the model
+    Parameters
+    ----------
+    config : AutoCompleteConfig object
+        object containing the model configuration
+    value : int
+        the max_batch_size we want to set for the model
+    """
+    config.set_max_batch_size(value)
+
+
+def set_input(config, new_input):
```

Comment:
Sorry for the late request. But what do you think about renaming `set_<property_name>` to `add_<property_name>`?
Also, new_input -> input
new_output -> output

Reply:
I think renaming `set_<property_name>` to `add_<property_name>` is more clear. But for max_batch_size, do we want to do the same?